sdk/textanalytics/ai-text-analytics/samples/v5/typescript/README.md

```diff
@@ -11,33 +11,33 @@ urlFragment: ai-text-analytics-typescript
+| [beginAnalyzeHealthcareEntities.ts][beginanalyzehealthcareentities]       | detects healthcare entities in a piece of text and prints them                                     |
+| [recognizePii.ts][recognizepii]                                           | detects personally-identifiable information                                                        |
 | [authenticationMethods.ts][authenticationmethods]                         | authenticates a service client using both Azure Active Directory and an API key                    |
-| [recognizeEntities.ts][recognizeentities]                                 | detects entites in a piece of text and prints them along with the entity type                      |
-| [beginAnalyzeHealthcareEntities.ts][beginanalyzehealthcareentities]       | detects healthcare entities of a piece of text                                                     |
-| [beginAnalyzeBatchActions.ts][beginanalyzebatchactions]                   | applies multiple Text Analytics actions per document                                               |
+| [beginAnalyzeBatchActions.ts][beginanalyzebatchactions]                   | extracts key phrases, entities, and pii entities from a piece of text                              |
+| [alternativeDocumentInput.ts][alternativedocumentinput]                   | uses object document inputs with attached metadata rather than simple strings for more flexibility |
+| [analyzeSentimentWithOpinionMining.ts][analyzesentimentwithopinionmining] | analyzes the sentiment of a piece of text and mine opinions about different targets                |
 
 ## Prerequisites
 
-The samples are compatible with Node.js >= 8.0.0.
+The sample programs are compatible with Node.js >= 12.0.0.
 
-Before running the samples in Node, they must be compiled to JavaScript using the TypeScript compiler. For more information on TypeScript, see the [TypeScript documentation][typescript]. Install the TypeScript compiler using
+Before running the samples in Node, they must be compiled to JavaScript using the TypeScript compiler. For more information on TypeScript, see the [TypeScript documentation][typescript]. Install the TypeScript compiler using:
 
 ```bash
 npm install -g typescript
 ```
 
-You need [an Azure subscription][freesub] and [an Azure Cognitive Services Instance][azcogsvc] to run these sample programs. Samples retrieve credentials to access the Cognitive Services endpoint from environment variables. Alternatively, edit the source code to include the appropriate credentials. See each individual sample for details on which environment variables/credentials it requires to function.
+You need [an Azure subscription][freesub] to run these sample programs. Samples retrieve credentials to access the endpoint from environment variables. Alternatively, edit the source code to include the appropriate credentials. See each individual sample for details on which environment variables/credentials it requires to function.
```

Comment:
do we care that we lost the link to creating the service instance here?

Reply:
Added a little config option and a system for rendering extra required Azure resources:

```javascript
{
  "//sampleConfiguration" : {
    ...,
    "requiredResources": {
      "Azure Cognitive Services": "<link to docs for creating it>"
    }
}
```